### ISSUE TITLE
Minor issue: Arduino IDE warning about 

### ISSUE THREAD

**Issue by pfeerick** [digistump/arduino-boards-index#2]
tl;dr Version number for bossac entry needs to be changed if you want to get rid of an annoying warning message in the log section every time you open the Arduino IDE Boards Manager. 

Have been getting an `Invalid version found: 1.3a-arduino` red text warning message from the Arduino IDE every time I open the Boards Manager for quite a while, and finally decided to do something about it. 

I knew it came from the the Digistump board manager JSON file, but not why. It appears that the version number parsing on the Arduino IDE Board Manager is very pedantic. The version number must follow some variant of the SemVer version number standard, or it gets cranky. 

So basically, the current '1.3a-arduino' does not work, and but something like '1.3-a-arduino' appears to... it seems that as long as there is a number after the dot, any text (or hyphen then text) is ignored?

> - If there are no dots (.), parse version as an integer and form a Version from that integer using Version.forIntegers
> - If there is one dot, split version into two, parse each part as an integer, and form a Version from those integers using Version.forIntegers
> - Otherwise, simply parse version into a Version using Version.valueOf

https://github.com/arduino/Arduino/wiki/Arduino-IDE-1.6.x-package_index.json-format-specification
